modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/controller/StaticResourceConfiguration.java

```diff
@@ -50,8 +64,12 @@ public StaticResourceConfiguration(ApplicationContext applicationCtx) {
         registry.addResourceHandler("/webjars/**")
             .addResourceLocations("classpath:/META-INF/resources/webjars/");
 
-        registry.addResourceHandler("/**").
-            addResourceLocations("file:frontend/");
+        if (frontendFolder != null) {
+            registry.addResourceHandler("/**")
+                .addResourceLocations(frontendFolder.toExternalForm());
+        }
+        else
+            log.info("Failed to find frontend resources locally: " + new File("frontend").getAbsolutePath());
```

Comment:
info -> warn

Reply:
I don't think this is a good idea, because we also have docker distributive without frontend resources.

Reply:
Improve message